docs/rules/prevent-abbreviations.md

```diff
@@ -0,0 +1,81 @@
+# Prevent abbreviations
+
+Using complete words results in more readable code. Not everyone knows all your abbreviations. You only write code once, but it's read many times.
+
+Default replacements are available [here](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/rules/prevent-abbreviations.js#L7).
+
+This rule is fixable for abbreviations with exactly one replacement defined.
+
+
```

Comment:
Can you add a short mention that this rule can be useful for more than just preventing abbreviations if they use the `extendDefaults` option. Could be used by companies to replace terms, disallowed words, etc.